internal/factsengine/gatherers/corosynccmapctl_test.go

```diff
@@ -0,0 +1,112 @@
+package gatherers
+
+import (
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	mocks "github.com/trento-project/agent/internal/factsengine/gatherers/mocks"
+)
+
+type CorosyncCmapctlTestSuite struct {
+	suite.Suite
+}
+
+func TestCorosyncCmapctlTestSuite(t *testing.T) {
+	suite.Run(t, new(CorosyncCmapctlTestSuite))
+}
+
```

Comment:
Maybe we will need to have a test for the scenario that the wanted fact is not present (new unhandled scenario I comment in a previous comment)